planner/core/rule_join_elimination.go

```diff
@@ -173,38 +183,66 @@ func (o *outerJoinEliminator) isDuplicateAgnosticAgg(p LogicalPlan) (_ bool, col
 			return false, nil
 		}
 		for _, expr := range aggDesc.Args {
-			if col, ok := expr.(*expression.Column); ok {
-				cols = append(cols, col)
-			} else {
-				return false, nil
-			}
+			// ExtractColumns will trans the expr to cols.
```

Comment:
This comment is useless, you can jump to the function to know it.

Reply:
Ok, I'll remove it.